openapi/components/paths/users.yaml

```diff
@@ -142,6 +157,22 @@ paths:
       x-internal: true
       deprecated: true
       description: '**REQUIRES ADMIN CREDENTIALS**. Search and list any Active users by text query.'
+  '/users/{userId}/groups/requested':
+    parameters:
+      - schema:
+          type: string
+        name: userId
+        in: path
+        required: true
+    get:
+      summary: Get User Guild Requests
+      tags:
+        - users
+      responses:
+        '200':
+          $ref: ../responses/groups/GroupListResponse.yaml
+      operationId: getGuildRequests
```

Comment:
"Guild"?

Reply:
Uh.. my stupidity 😅
Used to guild from discord too much.